Documentation/tutorial/validate your markdown files.md

```diff
@@ -150,3 +150,33 @@ public class MyMarkdownTokenValidatorProvider : IMarkdownTokenValidatorProvider
 ## How to enable custom validators
 1.  Config markdown validation rules in `md.style`.
 2.  Copy your extension assemblies to `plugin` folder in your `DocFX.exe` or template.
+
+## Validate metadata in markdown files
```

Comment:
I think this section should be placed before the ##How to enable section. and need to let user know that to enable this, the md.style file doesn't need to be configured..